lib/zlib.js

```diff
@@ -23,13 +23,21 @@
 
 const Buffer = require('buffer').Buffer;
 const Transform = require('_stream_transform');
+const { _extend } = require('util')._extend;
```

Comment:
@mscdex  what it looks like to me is if you are deconstructing the object, why do you have the `_extend` property? e.g. why not just `+const { _extend } = require('util')`